README.md

```diff
@@ -6,29 +6,65 @@ Builds of Mission Control can currently be found in the Oracle JDK on supported
 
 For more information on Mission Control, see http://www.oracle.com/missioncontrol.
 
-### Core API Features
+## Downloading Builds
+Binary distributions of JDK Mission Control are provided by different downstream vendors.
 
-* Core APIs for parsing and processing Java flight recordings 
+### AdoptOpenJDK
+* Released version
+* EA builds of upcoming release
+* Downloadable Eclipse update site archive
 
-* Core API can *read* recordings from JDK 7 and above
+[http://adoptopenjdk.net/jmc](http://adoptopenjdk.net/jmc)
 
-* Core API can *run* on JDK 7 and above
 
-* Core API contains a framework for handling units of measurement and physical quantities
+### Azul (Zulu Mission Control)
+* Released version
 
-* Core API supports headless analysis of Java flight recordings
+[https://www.azul.com/products/zulu-mission-control](https://www.azul.com/products/zulu-mission-control)
+
+
+### Bell-Soft (Liberica Mission Control)
+* Released version
+
+[https://bell-sw.com/downloads](https://bell-sw.com/downloads)
+
+### Oracle
+* Released version
+* Integrated (in-app) update site
+* Eclipse update site
+* Downloadable Eclipse update site archive
```

Comment:
Currently we are not delivering "Downloadable Eclipse update site archive" 
Please remove the above line.

Reply:
Ok!